eng/pipelines/coreclr/templates/perf-job.yml

```diff
@@ -66,6 +65,7 @@ jobs:
     iosStripSymbols: ${{ parameters.iosStripSymbols }}
     hybridGlobalization: ${{ parameters.hybridGlobalization }}
     timeoutInMinutes: ${{ parameters.timeoutInMinutes }}
+    extraSetupParameters: $(extraSetupParameters)
```

Comment:
Where is the extraSetupParameters coming from? Does the initialization below get replaced above/here.

Reply:
This variable is defined lower in the variables section, it does seem to work properly in my testing